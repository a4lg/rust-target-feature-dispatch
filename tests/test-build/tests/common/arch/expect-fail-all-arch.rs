pub const VALUE: i32 = target_feature_dispatch::target_feature_dispatch! {
    // SPECIFIC: ARCH_COMPILER_VARIANT
    if "aarch64" || "arm" || "arm64ec" || "avr" || "bpf" || "csky" || "hexagon" || "loongarch64" || "m68k" || "mips" || "mips32r6" || "mips64" || "mips64r6" || "msp430" || "nvptx64" || "powerpc" || "powerpc64" || "riscv32" || "riscv64" || "s390x" || "sparc" || "sparc64" || "wasm32" || "wasm64" || "x86" || "x86_64" || "xtensa" {
        {
            compile_error!("OK");
            1
        }
    } else {
        0
    }
};
fn main() {}
