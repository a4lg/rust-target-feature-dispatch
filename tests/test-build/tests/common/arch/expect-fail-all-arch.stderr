error: OK
 --> tests/common/arch/expect-fail-all-arch.rs:5:13
  |
5 |             compile_error!("OK");
  |             ^^^^^^^^^^^^^^^^^^^^
